src/java.management/share/classes/java/lang/management/MemoryMXBean.java

```diff
@@ -267,6 +267,24 @@ public interface MemoryMXBean extends PlatformManagedObject {
      */
     public MemoryUsage getNonHeapMemoryUsage();
 
+    /**
+     * Returns the CPU time used by all garbage collection threads.
+     *
+     * <p> This include time since genesis, so the value can be
+     * non-zero even if no garbage collection cycles occured. In
+     * general this includes time for all driver threads,
+     * workers, VM operations on the VM thread and the string
+     * deduplication thread (if enabled). This method returns
+     * {@code -1} if the platform does not support this operation
+     * or if called during shutdown.
+     *
+     * @return the total CPU time for all garbage collection
+     * threads in nanoseconds.
```

Comment:
Not sure did I miss the discussion, other methods like getTotalCompilationTime() return millis, is it ok or required to use new units here?